Question5_ArraySorted.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * This class is designed to check Sort Order of array
+ * @author Gaurav Tak
+ */
+public class Question5_ArraySorted {
+	/**
+	 * This method checks the sorting order for a given array
+	 * @param input
+	 * @return flag value for sorting order - 0:unsorted,1:ascending,2:descending 
+	 */
+	public int sortOrder(int[] input) {
+		int i=1;
+		int flag=0;
```

Comment:
It will fail for input array with size 0 and 1
How it will go if we use variables to store length of ascending and descending seq and check with length of array